openshift_images/managing_images/using-image-pull-secrets.adoc

```diff
@@ -8,6 +8,11 @@ toc::[]
 
 If you are using the {product-title} internal registry and are pulling from image streams located in the same project, then your pod service account should already have the correct permissions and no additional action should be required.
 
+[NOTE]
+====
+When you create a pod service account or a namespace, wait until it is provisioned with a docker pull secret; if you create a pod before its service account is fully provisioned, the pod fails to access the {product-title} internal registry.
```

Comment:
wait until the service account is provisioned with a docker pull secret

Reply:
@xiuwang Comment addressed.